src/internal.h

```diff
@@ -154,6 +154,8 @@
 #define DISPATCH_PURE_C 1
 #endif
 
+#include <pthread.h>
```

Comment:
oh I see, then I would move that `#include` right after this comment:

```
/* private.h must be included last to avoid picking up installed headers. */
```